openssl/src/x509/mod.rs

```diff
@@ -649,6 +649,22 @@ impl X509Ref {
         }
     }
 
+    /// Returns this certificate's "alias". This field is populated by
+    /// OpenSSL in some situations -- specifically OpenSSL will store a
+    /// PKCS#12 `friendlyName` in this field.
```

Comment:
I might suggest adding a sentence that this is not part of the certificate itself, just some auxiliary info that OpenSSL staples onto it. In particular, it doesn't get serialized. It's just a way to get the friendly name out of you used pkcs12 APIs.

Reply:
Yeah, I should make that explicit rather than implicit.